packages/calling-stateful-client/review/calling-stateful-client.api.md

```diff
@@ -201,6 +203,12 @@ export interface TranscriptionCallFeature {
     isTranscriptionActive: boolean;
 }
 
+// @public (undocumented)
+export type VideoStreamRendererAndView = {
```

Comment:
I'm uncomfortable naming a type for the fields it contains (can never add a field). `CreateViewResult` would be a better name.

Reply:
Good point, yeah I agree, updated